packages/@vue/cli/lib/Creator.js

```diff
@@ -388,23 +389,36 @@ module.exports = class Creator extends EventEmitter {
 
     // ask for packageManager once
     const savedOptions = loadOptions()
-    if (!savedOptions.packageManager && hasYarn()) {
+    if (!savedOptions.packageManager && (hasYarn() || hasPnpm())) {
+      const packageManagerChoices = []
+
+      if (hasYarn()) {
+        packageManagerChoices.push({
+          name: 'Use Yarn',
+          value: 'yarn',
+          short: 'Yarn'
+        })
+      }
+
+      packageManagerChoices.push({
```

Comment:
What do you think to move npm choice at the end of choices?
It will increase code readability and also keep some consistency with this _pattern_: 
```js
if (hasYarn()) {
  // Do Yarn thing
}

if (hasPnpm()) {
  // Do PNPM thing
}

// Do NPM thing
```